linux/Platform.h

```diff
@@ -72,7 +72,7 @@ bool Platform_getDiskIO(DiskIOData* data);
 
 bool Platform_getNetworkIO(NetworkIOData* data);
 
-void Platform_getBattery(double *percent, ACPresence *isOnAC);
+void Platform_getBattery(Meter* this, double* percent, ACPresence* isOnAC);
```

Comment:
It's arguably better to have `T *t` instead of `T* t`, because when you define multiple variables of the same type in one go you would write `T *t1, *t2`, not `T* t1, t2`. Regardless, such changes are out-of-place for this commit.

Reply:
Please note the general coding style, which seems to prefer `T* t`.

Reply:
I see. Please dismiss this then. :)